x-pack/plugins/elastic_assistant/common/constants.ts

```diff
@@ -12,6 +12,9 @@ export const BASE_PATH = '/internal/elastic_assistant';
 
 export const POST_ACTIONS_CONNECTOR_EXECUTE = `${BASE_PATH}/actions/connector/{connectorId}/_execute`;
 
+// Insights
+export const INSIGHTS_ALERTS = `${BASE_PATH}/insights/alerts`;
```

Comment:
A little curious that this is not the same path used elsewhere for insights, `ai_insights`